packages/web/src/utils/route.ts

```diff
@@ -541,3 +544,24 @@ export const pushUniqueRoute = (location: Location, route: string) => {
   }
   return { type: '' }
 }
+
+export const getSearchPageLocation = ({
+  category,
+  ...searchParams
+}: {
+  category?: 'all' | 'tracks' | 'profiles' | 'albums' | 'playlists'
+  query?: string
+  genre?: Genre
+  mood?: Mood
+  bpm?: string
+  key?: string
+  // TODO: Not needed yet, add when used
```

Comment:
mind including the hasDownloads now that is merged. isVerified and isPremium not currently active?

Reply:
We dont have any links that would use these currently, but I can update now to make all of them work

Reply:
sweet lmk if any of the new api values dont work as expected